mcbackend/test_adapter_pymc.py

```diff
@@ -87,23 +87,21 @@ def wrapper(meta: RunMeta):
             trace = TraceBackend(backend)
             idata = pm.sample(
                 trace=trace,
-                tune=3,
-                draws=5,
+                tune=30,
+                draws=50,
```

Comment:
is the rng set somewhere to guarantee a warning?

Reply:
within these steps

Reply:
no, but with this few steps the NUTS is pretty much guaranteed to emit warnings.
And even if it occasionally does not, that should not break things.